pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
-                    page2 = None
-                    while tryCounter <= tryLimit:
-                        try:
-                            if self.debug_mode >= 1:
-                                print("Try: ", tryCounter, " ", URL)
-                            page = requests.get(URL)
+                # Stop the runloop for this process
+                self.need_quit = True
 
-                            if page.status_code != 200:
-                                raise Exception("Error Code: ", page.status_code)
+            except Exception as error:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)